runner/src/main.rs

```diff
@@ -47,6 +47,11 @@ struct RunArgs {
     #[arg(long)]
     /// Path to the configuration file to use
     config: Option<PathBuf>,
+    #[arg(short, long, action)]
+    benchmark: bool,
+    /// Type of benchmark, default is instr
+    #[arg(long, default_value = "instr")]
+    bench_type: String,
```

Comment:
What if we want to measure multiple things at once? Technically RISC-V allows to use multiple counters simultaneously, like instructions retired + elapsed time.

It's probably something we should take into account, it's often useful to have multiple metrics simultaneously when analyzing performances. A typical example would be cache hit + cache misses, both are needed to draw conclusion about the hit ratio, and that is what tools such as `perf` do.

Reply:
I was thinking to reduce benchmark influence on each other. If we want to measure number of instruction, we don't want to count instruction of the benchmark code. But it will be a fixed amount of difference for each measurement that we can subtract somehow, measuring only the delta, so I can merge everything.